tests/utils/connection/connection.go

```diff
@@ -0,0 +1,74 @@
+package connection
+
+/*
+Copyright (c) 2018 Red Hat, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+import (
+	"fmt"
+
+	. "github.com/onsi/ginkgo/v2"
+
+	client "github.com/openshift-online/ocm-sdk-go"
+)
+
+// Regular users in the organization 'Red Hat-Service Delivery-tester'
+var (
+	RosaConnection = createConnectionWithToken(RHCSOCMToken)
+)
+
+var (
+	// Create a logger:
+	logger = createLogger()
+)
+
+func createConnectionWithToken(token string) *client.Connection {
```

Comment:
Let's keep the copied code for now. In future backend will be miraged to github, then it can be imported

Reply:
Does it mean no changes needed currently?

Reply:
Yes